src/app/components/Layout/Layout.jsx

```diff
@@ -162,6 +163,25 @@ const Layout = () => {
     }
   }
 
+  const onAutoUpdateError = () => {
+    addToast({
+      id: 'auto-update-error',
+      title: 'Auto-Update Failure',
+      message: 'Failed to download latest update. Download new version manually:',
```

Comment:
```suggestion
      message: 'Failed to download latest update. Download new version manually',
```
Do we want that colon at the end there?